uninstall.php

```diff
@@ -17,6 +17,8 @@
 	define( 'JETPACK__PLUGIN_DIR', plugin_dir_path( __FILE__ )  );
 }
 
+$jetpack_autoloader = JETPACK__PLUGIN_DIR . 'vendor/autoload_packages.php';
```

Comment:
Missing a `require_once` here? Or am I misunderstanding this bug?

Reply:
it is missing a require once or classes registered for autoload, but we are using the Sender further down so I guess it is safer to register the entire Jetpack class map

Reply:
Right but... how is it loading the classes? You are just defining `$jetpack_autoloader` as a string

Reply:
hehe, you win